go/vt/vttablet/endtoend/misc_test.go

```diff
@@ -1090,6 +1090,30 @@ func TestEngineReload(t *testing.T) {
 	})
 }
 
+func TestUpdateTableIndexMetrics(t *testing.T) {
```

Comment:
do we have tables with partitions in this e2e test. 
Is it possible to validate the value than checking for nil?

Reply:
Updated the tests to explicitly test values, rather than just assert not-nil